.github/workflows/scripts/helm-weekly-release.sh

```diff
@@ -42,6 +42,17 @@ increment_semver() {
   echo $(local IFS=$delimiter ; echo "${array[*]}")
 }
 
+# Sets a part of the semver string
+# $1: version itself
+# $2: number of part: 0 – major, 1 – minor, 2 – patch
+# $3: the number to set the part to
+set_semver() {
```

Comment:
It seems the only time we use this only to set the patch version to 0 of the current version. Therefore we can simplify this function to just accept the first argument. The function name also can be updated to reflect this.

I.e

```
set_patch_version_zero "1.2.3" # 1.2.0
set_patch_version_zero "2.2.0" # 2.2.0
```